.github/workflows/tests_secondary.yml

```diff
@@ -535,7 +535,7 @@ jobs:
       shell: bash
       env:
         PWTEST_CHANNEL: msedge-beta
-        PWTEST_BLOB_SUFFIX: "-edge-beta-windows"
+        PWTEST_BLOB_REPORT_NAME: "-edge-beta-windows"
```

Comment:
Let's drop leading dash from '-edge...' and in other places too. We can prepend it where needed in html report and in the blob file name but it's a bit odd to have report-name always start with `-`.